src/sestest/VerifyCommand.cs

```diff
@@ -54,31 +62,54 @@ public VerifyCommand(ILogger logger) : base(logger)
 
         private async Task<string> SubmitToken(Uri server, string token, string app, string api)
         {
-            var builder = new UriBuilder(server);
-            //TODO: Do this in a path safe way similar to Path.Combine
-            builder.Path = builder.Path + "softwareEntitlements";
-            builder.Query = CreateParameters(("api-version", api));
+            var softwareEntitlementsUri = new Uri(server, "softwareEntitlements");
```

Comment:
Does this form work with test style batch URLs?

Production looks like `https://account.region.batch.azure.com` 
Test looks like `https://testtenant.batch.azure.com/account`

Reply:
Yuk. Not really. Only if the test URL ends with a `/`:
`new Uri(new Uri("https://testtenant.batch.azure.com/account"), "softwareEntitlements")` => `"https://testtenant.batch.azure.com/softwareEntitlements"`
`new Uri(new Uri("https://testtenant.batch.azure.com/account/"), "softwareEntitlements")` => `"https://testtenant.batch.azure.com/account/softwareEntitlements"`

Do you think we should just go back to string concatenation and checking for the presence of a trailing slash? Or any ideas for framework methods that might work better?